roles/cadvisor/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 # tasks for running cadvisor
+- name: destroy old cadvisor container
+  when: cadvisor_rebuild_container
```

Comment:
Not only for this PR we should set a convention about using "| bool" filter or not with booleans as we have both cases now. The filter is safer in terms of a value passing though via automation e.g export variable used by bash script used by ansible